### DISCUSSION TITLE
캐릭터 얼굴 표정 애니메이팅은 불가능한가요?

### DISCUSSION THREAD

**Discussion by hhnmsLeesan** [naverz/zepeto-studio-kor#568] [5. Q&A]
안녕하세요!

사진을 찍어 유저의 제페토 피드에 업로드하는 기능이 활성화되어 그것을 활용한 다양한 컨텐츠를 기획 및 개발하고자 합니다.
때문에 제페토 캐릭터의 표정이나 포즈를 다양하게 변화시켜 보고자 하는데요.

이 과정에서 얼굴 bone에 해당하는 GameObject를 건드려도 이목구비의 크기에는 변화가 생겨도 표정에는 변화가 없다는 사실을 발견했습니다. 리깅이 아예 되어 있지 않은 것으로 추측하는데요.

제페토 공식 월드를 보면 제페토 캐릭터들이 다양한 제스처를 취하면서 표정도 함께 변화하는 것이 확인되는데, 혹시 유니티-제페토에서도 이렇게 표정까지 변경할 수 있도록 지원하실 계획이 향후에 있으신지 궁금합니다.

감사합니다!

**Comment by x4cf** [comment]
표정변화 애니메이션은 현재 제작 가능합니다 :D

**Reply by hhnmsLeesan** [reply]
공식 개발자가 아니면 답변을 자제해주셨으면 좋겠습니다.
이왕 답변을 달아주실 거라면 답변을 구체적으로 달아주시는 것이 서로 도움이 될 것 같습니다.
